test/csuite/timestamp_abort/main.c

```diff
@@ -749,43 +749,34 @@ main(int argc, char *argv[])
     uint64_t commit_fp, durable_fp, stable_val;
     uint32_t i, timeout;
     int ch, status, ret;
-    const char *working_dir;
     char buf[512], fname[64], kname[64], statname[1024];
     char ts_string[WT_TS_HEX_STRING_SIZE];
-    bool fatal, preserve, rand_th, rand_time, verify_only;
+    bool fatal, rand_th, rand_time, verify_only;
 
     (void)testutil_set_progname(argv);
 
-    columns = compat = inmem = stress = false;
+    buf[0] = '\0';
+    opts = &_opts;
+    memset(opts, 0, sizeof(*opts));
+
+    columns = stress = false;
     use_ts = true;
     nth = MIN_TH;
-    preserve = false;
     rand_th = rand_time = true;
     timeout = MIN_TIME;
     verify_only = false;
-    working_dir = "WT_TEST.timestamp-abort";
 
-    while ((ch = __wt_getopt(progname, argc, argv, "Cch:LmpsT:t:vz")) != EOF)
+    testutil_parse_begin_opt(argc, argv, "b:CmPTh:pv", opts);
+
+    while ((ch = __wt_getopt(progname, argc, argv, "Cch:LmpsP:T:t:vz")) != EOF)
```

Comment:
In the first (`testutil_parse_begin_opt`) list, I think you need a `:` after `P`.   I see you have `v` in the first list, but it is parsed here in main.  That's because `'v'` stands for verify in this program not verbose.  So you should remove `v` from the first list.  Same with `T:`, it's being handled by the main program, so it should be removed from the first list.  Same with `t:`, it's handled by the main program (and has a different meaning).

I think you need the `b:` in the second list.

I have to say, it's a bit confusing keeping these two lists straight.  I have an idea about streamlining this, I'll put in another ticket.

Reply:
Thanks for the suggestions. Fixed now.